maintenance_timesheet/models/hr_timesheet.py

```diff
@@ -16,11 +16,15 @@ def onchange_maintenance_request_id(self):
             self.project_id = self.maintenance_request_id.project_id
             self.task_id = self.maintenance_request_id.task_id
 
-    @api.model
-    def create(self, values):
-        if values.get("maintenance_request_id"):
-            self._check_request_done(values.get("maintenance_request_id"))
-        return super().create(values)
+    @api.model_create_multi
+    def create(self, vals_list):
+        maintenance_request_ids = [
+            vals.get("maintenance_request_id")
+            for vals in vals_list
+            if vals.get("maintenance_request_id")
+        ]
+        self._check_requests_done(maintenance_request_ids)
+        return super().create(vals_list)
```

Comment:
Please don't complicate API with 2 methods with just an extra `s` (very easy to be typo error), and don't complicate logic. It's easier this way:

```suggestion
        for vals in vals_list:
            if vals.get("maintenance_request_id"):
                self._check_requests_done(vals["maintenance_request_id"])
        return super().create(vals_list)
```

Reply:
@IT-Ideas We could keep the same method and document that it could be called with a single id or with a list of ids...